src/metaswitch/ellis/test/api/users.py

```diff
@@ -320,7 +320,10 @@ def save(self, email, token, password):
         self.handler.post(EMAIL)
 
         # Should succeed, but set the wrong password.
-        set_recovered_password.assert_called_once()
+        set_recovered_password.assert_called_with(ANY,
+                                                  ANY,
+                                                  ANY,
+                                                  ANY)
```

Comment:
I now guess that these changes are related to taking an upgraded version of one of our dependencies, if so, fine.